src/drivers/libretro/libretro.c

```diff
@@ -1347,12 +1362,16 @@ static void check_variables(bool startup)
 
    if (environ_cb(RETRO_ENVIRONMENT_GET_VARIABLE, &var) && var.value)
    {
-      bool newval = (!strcmp(var.value, "8:7 PAR"));
-      if (newval != use_par)
-      {
-         use_par = newval;
+     last_aspect_ratio_par = aspect_ratio_par;
```

Comment:
make **last_aspect_ratio_par** local since its not used anywhere outside of this function.